Modules/_ctypes/_ctypes.c

```diff
@@ -2244,10 +2245,16 @@ PyCSimpleType_from_param(PyObject *type, PyObject *value)
         value = PyCSimpleType_from_param(type, as_parameter);
         _Py_LeaveRecursiveCall();
         Py_DECREF(as_parameter);
+        Py_XDECREF(exc);
+        Py_XDECREF(val);
+        Py_XDECREF(tb);
```

Comment:
Doesn't this need to be before line 2243, otherwise you might leak these references.

Reply:
Yes, I think you're right. I pushed an update. Thanks! 🙌

Reply:
It is also required on line 2238